be/src/exec/vectorized/jni_scanner.cpp

```diff
@@ -0,0 +1,333 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "jni_scanner.h"
+
+#include "column/type_traits.h"
+#include "fmt/core.h"
+#include "util/defer_op.h"
+
+namespace starrocks::vectorized {
+
+Status JniScanner::reset_jni_env() {
+    _jni_env = JVMFunctionHelper::getInstance().getEnv();
```

Comment:
why we need acquire _jni_env

Reply:
to wrap JNI method calls

Reply:
I think it is unnecessary

Reply:
could you provide more details? maybe some gaps exist in our thoughts

Reply:
done